demos/serverless/deploy.js

```diff
@@ -20,8 +41,11 @@ function usage() {
   console.log(`  -a, --application                    Browser application to deploy, default '${app}'`);
   console.log(`  -e, --event-bridge                   Enable EventBridge integration, default is no integration`);
   console.log(`  -c, --chime-endpoint                 AWS SDK Chime endpoint, default is '${chimeEndpoint}'`);
+  console.log(`  -p, --service-principal              Service principal for meeting related resources, default is '${chimeServicePrincipal}'`)
   console.log(`  -t, --enable-termination-protection  Enable termination protection for the Cloudformation stack, default is false`);
   console.log(`  -l, --disable-printing-logs          Disable printing logs`);
+  console.log(`  -o, --capture-output-prefix          Prefix for S3 bucket name`);
+  console.log(`  -i, --opt-in-regions                 Comma separated list of additional opt-in regions to enable for media capture`);
```

Comment:
Not familiar with capturing so could you explain more about this option?

Reply:
The `-o` option is used for specifying the name of the S3 buckets.  It's a prefix because we append the region since we need a bucket in each region.
The `-i` option allows to specify additional regions beyond the initial set.   This is because we support meetings in `af-south-1` and `eu-south-1` which are opt-in and require customers to enable those regions in their AWS accounts before they can be used.  If customers want media capture in those regions they need to specify them here to create the S3 buckets.

Reply:
Could we add notes about this in the README?